src/components/ui/Chat/MessageAttachment/Styled.tsx

```diff
@@ -30,9 +33,9 @@ export const StyledMessageAttachmentContent = styled.div`
     margin-left: 1rem;
 
     & a:link,
-    a:visited,
-    a:hover,
-    a:active {
+    & a:visited,
+    & a:hover,
+    & a:active {
```

Comment:
This is to handle https://styled-components.com/docs/faqs#nested-syntax-handling